### ISSUE TITLE
General bug with mobs going through caverns

### ISSUE THREAD

**Issue by SASzou** [IcecrownServer/Icecrown#153]
Bug: Some mobs that you find on the surface of a zone are found in the caverns of that same zone. Examples below.

How it should work: The mobs on the surface should remain on the surface and not be found down below.

How to reproduce: Example: Go into hillsbrad foothills caverns like Azurelode Mine or the Yeti cave and check out the mobs in there

Source: general knowledge of the game
Here are some examples:
![cavern1](https://cloud.githubusercontent.com/assets/25365123/23342970/4f8443ce-fc6c-11e6-9d7b-b3b5049ab6c3.jpg)
![cavern2](https://cloud.githubusercontent.com/assets/25365123/23342976/5c179578-fc6c-11e6-9fd6-82425d9e2416.jpg)
![cavern3](https://cloud.githubusercontent.com/assets/25365123/23342979/63af3872-fc6c-11e6-9c6c-3fd01d592f28.jpg)

**Comment by warcraftgeekygamer** [comment]
Yes, I also experienced this very same issue while questing in Hillsbrad Foothills. You are unable to attack the mobs that have fallen through the ground and into the cave below as they are evade bugged.
